### ISSUE TITLE
Retraining with custom pretrained model

### ISSUE THREAD

**Issue by akashjborah97** [ArgoHA/D-FINE-seg#46]
I have added new data & fixed some old data in my dataset. I want to start a new training using my previous baseline. Are there any parameters in config.yaml file that I need to change? Do I need to change "clip_max_norm" parameter & how?

**Comment by ArgoHA** [comment]
It does `torch.nn.utils.clip_grad_norm_(self.model.parameters(), self.clip_max_norm)`. So it clips gradients before doing optimizer.step(), so training is more stable and you don't get blown gradients. Although it still can happen with higher LRs.
